### PR TITLE
Update endpoint-as2-receive.adoc

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: modules/ROOT/pages/configure-partner.adoc [hunk 1]
```diff
@@ -128,7 +130,7 @@ Remember that there must always be at least one identifier per message flow befo
 
 After successfully deleting an identifier, you can create a new identifier with the same value as the one you deleted.
 
-You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identifiers.
+You can delete identifiers such as all X12 ISA, X12 GS, AS2, and DUNS identifiers. You cannot delete the last identifier.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: I don't understand the second sentence in light of the first, which says you can delete ALL identifiers.
And what is "the last identifier"? The last of each type in a type list, or the last listed for all types, or...?

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 2]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
 Required: No +
 
-| AS2 URL (Automatically generated at time of deployment)
+| AS2 URL
 | The HTTP or HTTPS address where you receive AS2 messages.
-The port is automatically generated when you deploy a message flow that uses this endpoint.
-You can see it by selecting the receiving endpoint from your deployed message flow.
-The hostname displayed is created by your administrator at setup.
 
+When deploying to CloudHub with a public Shared Load Balancer, the URL is automatically generated when you deploy a message flow that uses this endpoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: unclear reference: the URL is not deploying....
```suggestion
The URL is automatically generated when you use a shared load balancer to deploy a message flow to CloudHub.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 3]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
 
-| Message Signature Encryption
-| If your partner sends an encrypted signature in the message, choose the appropriate algorithm for validating that signature: +
+To get the external facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub with Dedicated load balancer) refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
To get the external-facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub by using a dedicated load balancer), refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer.
```
...but do we really want to introduce a goal and then send the audience off to some other source to tell how to achieve that goal?

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 4]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
 
-Options are: +
+You view it by selecting the receiving endpoint from your deployed message flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: View what?

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 5]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
-`AES256_WRAP` +
-`CAST5` +
-`IDEA` +
-`RC2_CBC` +
-
-Required: Yes +
-
-| Message Subject
-| Specify a default subject for messages you send to this partner +
+If deploying to On-premises Mule with a Firewall and Load Balancer, work with your hosting or Network Security team to obtain the externally accessible DNS /host name where the Mule runtime server is executing, and replace {runtime_host} in the URL. For example, if the DNS host name is b2b.mycompany.com - the AS2 Receiver URL is: https://b2b.mycompany.com/receive-as2/
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
If deploying to on-premises Mule with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule runtime server is executing, and replace {runtime_host} in the URL. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is `https://b2b.mycompany.com/receive-as2/`.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 6]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
 
 Required: Yes +
 
-| Compress
-| Indicates if the trading partner wants to see the incoming message and associated signature in compressed form.
-Zlib is used for compression. +
-Required: No +
-Default: False +
+| Key passphrase
+|  This is the passphrase for your private key
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
| This is the passphrase for your private key.
```

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 7]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
-|The Sending partner’s email used in AS2 transmissions. +
 Required: Yes +
 
-| Partner to email
-|The Receiving partner’s email used in AS2 transmissions. +
-Required: Yes +
+= Supported Signing and Encryption Algorithms
 
-|===
+AS2 receive endpoints deployed from Partner Manager support the below signing & encryption algorithms.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]:
```suggestion
AS2 receive endpoints deployed from Partner Manager support the following signing and encryption algorithms.
```

**Review by kbostrom — APPROVED** [review]
Copyedited

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: This text has been removed as it is no longer accurate.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: It's a set of steps only related to load balancer so yes, I think so.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: Fixed.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: Fixed.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 7 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: REmoved.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 2 above)
Comment (vijayang) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: We do not want to bring the topic of load balancer at all. 
We just need to have the below with respect to CloudHub.

`When deploying to CloudHub the URL is automatically generated when you deploy a message flow that uses this endpoint. You can see it by selecting the receiving endpoint from your deployed message flow.`

File: modules/ROOT/pages/configure-partner.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: REworded.

File: modules/ROOT/pages/endpoint-as2-receive.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoint-as2-receive.adoc]: Fixed.

File: modules/ROOT/pages/endpoint-as2-receive.adoc [hunk 8]
```diff
@@ -13,105 +13,33 @@ Required: Yes +
...
-The port is automatically generated when you deploy a message flow that uses this endpoint.
-You can see it by selecting the receiving endpoint from your deployed message flow.
-The hostname displayed is created by your administrator at setup.
 
+When deploying to CloudHub the URL is automatically generated when you deploy a message flow that uses this endpoint.
+You can see the URL  by selecting the receiving endpoint from your deployed message flow.
 
-| Message Signature Encryption
-| If your partner sends an encrypted signature in the message, choose the appropriate algorithm for validating that signature: +
+To get the external-facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub with Dedicated load balancer) refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-receive.adoc]: This sentence needs to be removed.

**Review by vijayang — CHANGES_REQUESTED** [review]

**Review by vijayang — APPROVED** [review]

**Pull request merged**